space_age.py

```diff
@@ -1,37 +1,44 @@
+SECONDS_IN_EARTH_YEAR = 60 * 60 * 24 * 365
+PLANETS_EARTH_YEAR_LENGTH = {
+    'Earth': 1,
+    'Mercury': 0.2408467,
+    'Venus': 0.61519726,
+    'Mars': 1.8808158,
+    'Jupiter': 11.862615,
+    'Saturn': 29.447498,
+    'Uranus': 84.016846,
+    'Neptune': 164.79132,
+}
+
 class SpaceAge(object):
-    SECONDS_IN_EARTH_YEAR = 60 * 60 * 24 * 365
-	
+
     def __init__(self, seconds):
         self.seconds = seconds
-        self.planets = {
-            'Earth': 1, 'Mercury': 0.2408467, 'Venus': 0.61519726,
-            'Mars': 1.8808158, 'Jupiter': 11.862615, 'Saturn': 29.447498,
-            'Uranus': 84.016846, 'Neptune': 164.79132
-        }
-        
+        years = self.seconds / SECONDS_IN_EARTH_YEAR
+        self.planets_age = {k: years * v for (k, v) in PLANETS_EARTH_YEAR_LENGTH.items()}
```

Comment:
@zarniak Please mind that dict comprehension syntax runs only in Python 3. In Python 2 that would need to be done "manually":
```
self.planets_age = {}
for k, v in PLANETS_EARTH_YEAR_LENGTH.items():
    self.planets_age[k] = years * v
```